test/typescript-lint/tsconfig.json

```diff
@@ -69,6 +69,9 @@
         "../../code/modules/data.d.ts",
         "../../code/modules/data.src.d.ts",
 
+        "../../code/modules/data-tools.d.ts",
+        "../../code/modules/data-tools.src.d.ts",
```

Comment:
This module is not part of distribution, only for testing purposes until Highcharts has data layer integration. So you can remove it.